api/views.py

```diff
@@ -54,3 +55,43 @@ class AuthorizedUserAccountsAPIView(ListAPIView):
 
     def get_queryset(self):
         return Account.objects.filter(user=self.request.user)
+
+
+class CreateAccountAPIView(CreateAPIView):
+    queryset = Account.objects.all()
+    serializer_class = AccountSerializer
+
+    authentication_classes = [TokenAuthentication]
+    permission_classes = [IsAuthenticated]
+
+    def perform_create(self, serializer):
+        serializer.save(user=self.request.user)
+
+
+class UpdateAccountAPIView(UpdateAPIView):
+    queryset = Account.objects.all()
+    serializer_class = AccountSerializer
+    lookup_field = 'number'
+
+
+class DeleteAccountAPIView(DestroyAPIView):
```

Comment:
1. combine list and create apis in one view and api url
2. combine detail, update and destroy in one view and api url